modules/alloy/src/main/ts/ephox/alloy/positioning/layout/Layout.ts

```diff
@@ -38,15 +38,18 @@ const southeast: AnchorLayout = (anchor: AnchorBox, element: AnchorElement, bubb
   bubbles.southeast(),
   Direction.southeast(),
   boundsRestriction(anchor, { left: AnchorBoxBounds.LeftEdge, top: AnchorBoxBounds.BottomEdge }),
-  'layout-se');
+  'layout-se',
+  false
```

Comment:
Find something better than a boolean.

Also, verify this is still required.

Reply:
Could make this a predicate that gets passed sizeInBounds and originInBounds for example.